File: Python/6kyu/is_pangram.py
# Detect Pangram
# https://www.codewars.com/kata/545cedaa9943f7fe7b000048

# My code
def is_pangram(s):
    char = 'a b c d e f g h i j k l m n o p q r s t u v w x y z'.split(' ')
    for i in char:
        if (i not in s.lower()): return False
    return True

# Best code
# import string
# def is_pangram(s):
#     return set(string.lowercase) <= set(s.lower())

File: JavaScript/6kyu/isPangram.js
// Detect Pangram
// https://www.codewars.com/kata/545cedaa9943f7fe7b000048

// My code
function isPangram(string){
  for (let i = 97; i <= 122; i++){
    const char = String.fromCharCode(i);
    if (string.toLowerCase().indexOf(char) === -1) return false;
  }
  return true;
}

// Best code
// function isPangram(string){
//   return 'abcdefghijklmnopqrstuvwxyz'
//     .split('')
//     .every((x) => string.toLowerCase().includes(x));
// }